project_euler/Problem 01/sol1.c

```diff
@@ -1,19 +1,24 @@
-/*
-If we list all the natural numbers below 10 that are multiples of 3 or 5, 
-we get 3, 5, 6 and 9. The sum of these multiples is 23.
-Find the sum of all the multiples of 3 or 5 below 1000.
-*/
+/*An Efficient code to print all the sum of all numbers that are multiples of 3 & 5 below N.*/
+
 #include <stdio.h>
 
-int main() {
-	int n = 0;
-	int sum = 0;
-	scanf("%d", &n);
-	for (int a = 0; a < n; a++) {
-		if ((a % 3 == 0) || (a % 5 == 0)) {
-			sum += a;
-		}
-	}
+int main(){
+    int t; 
+    scanf("%d",&t);
```

Comment:
This line is awkward. Can you write a ```while``` condition such that the user can exit the program bei typing -1 ?
Otherwise you can remove the full while-loop, because the problem definition is with your efficient solution solve.

Reply:
The question mentioned that the First Line contains an Integer T which denotes the Number of test cases. Hence the above lines.

Reply:
You can put in while condition for breaking. For example if the user type ```-1``` the program terminates.